client/src/components/EditVolunteerPopup.tsx

```diff
@@ -46,12 +47,14 @@ const EditFieldInput = ({
 };
 
 const participantTypes: string[] = ["Driver", "Packer", "Driver & Packer"];
+//const timeSlots: string[]= ["9:30", "10:30"]
```

Comment:
you can remove commented out code lines. no need to keep and merge those into the codebase.